extensions/typescript-language-features/extension-browser.webpack.config.js

```diff
@@ -60,30 +57,20 @@ module.exports = withBrowserDefaults({
 				}))
 			],
 		}),
-		// @ts-ignore
-		new CopyPlugin({
-			patterns: [
-				{
-					from: '../node_modules/typescript/lib/tsserver.js',
-					to: 'typescript/tsserver.web.js',
-					transform: async (content) => {
-						const dynamicImportCompatPath = path.join(__dirname, '..', 'node_modules', 'typescript', 'lib', 'dynamicImportCompat.js');
-						const prefix = fs.existsSync(dynamicImportCompatPath) ? fs.readFileSync(dynamicImportCompatPath) : undefined;
-						const output = await Terser.minify(content.toString());
-						if (!output.code) {
-							throw new Error('Terser returned undefined code');
-						}
-
-						if (prefix) {
-							return prefix.toString() + '\n' + output.code;
-						}
-						return output.code;
-					},
-					transformPath: (targetPath) => {
-						return targetPath.replace('tsserver.js', 'tsserver.web.js');
-					}
-				}
-			],
-		}),
 	],
-});
+}), withBrowserDefaults({
+	context: __dirname,
+	entry: {
+		'typescript/tsserver.web': './web/webServer.ts'
+	},
+	output: {
+		// all output goes into `dist`.
+		// packaging depends on that and this must always be like it
+		filename: '[name].js',
+		path: path.join(__dirname, 'dist', 'browser'),
+		libraryTarget: undefined,
```

Comment:
withBrowserDefaults sets `libraryTarget: commonjs`, which is not correct for tsserver.web.js. I don't know if it's correct for any callers, but there are lots of them so I didn't change it.

Reply:
It's correct for all callers because they should all be extensions, which even on web are CJS (they are executed in a funky eval environment).